angular2-modal: EXCEPTION: Uncaught (in promise): undefined

I get the following error when I close the modal alert:

```
core.umd.js:6001 Error: Uncaught (in promise): undefined
    at resolvePromise (zone.js:418)
    at PromiseCompleter.reject (zone.js:395)
    at _dismiss (angular2-modal.umd.js:391)
    at eval (angular2-modal.umd.js:394)
    at ZoneDelegate.invoke (zone.js:192)
    at Object.onInvoke (core.umd.js:8772)
    at ZoneDelegate.invoke (zone.js:191)
    at Zone.run (zone.js:85)
    at zone.js:451
    at ZoneDelegate.invokeTask (zone.js:225)
```

Sample code:
```
       this.modal.alert()
            .size("lg")
            .showClose(true)
            .title("title")
            .body("message")
            .open();

```
Dependencies:
```
"dependencies": {
    "@angular/common": "2.0.0-rc.6",
    "@angular/compiler": "2.0.0-rc.6",
    "@angular/compiler-cli": "0.6.0",
    "@angular/core": "2.0.0-rc.6",
    "@angular/forms": "2.0.0-rc.6",
    "@angular/http": "2.0.0-rc.6",
    "@angular/platform-browser": "2.0.0-rc.6",
    "@angular/platform-browser-dynamic": "2.0.0-rc.6",
    "@angular/router": "3.0.0-rc.2",
    "@angular/upgrade": "2.0.0-rc.6",
    "core-js": "^2.4.1",
    "reflect-metadata": "^0.1.3",
    "rxjs": "5.0.0-beta.11",
    "systemjs": "0.19.27",
    "zone.js": "^0.6.17",
    "angular2-in-memory-web-api": "0.0.18",

    "angular2-modal": "^2.0.0-beta.10",
    "bootstrap": "^3.3.7",
    "font-awesome": "^4.6.3",
    "jquery": "^3.1.0",
    "momentjs": "^1.1.15",
    "ng2-translate": "^2.4.4"
  },
```

--- a/src/components/angular2-modal/models/dialog-ref.ts
+++ b/src/components/angular2-modal/models/dialog-ref.ts
@@ -79,6 +79,7 @@ export class DialogRef<T> {
   dismiss() {
     const _dismiss = () => {
       this.destroy();
+      this._resultDeferred.promise.catch(() => {});
       this._resultDeferred.reject();
     };
     this._fireHook<boolean>('beforeDismiss')

